code/datums/components/crafting/tailoring.dm

```diff
@@ -317,3 +317,17 @@
 	)
 
 	category = CAT_CLOTHING
+	
+	
+/datum/crafting_recipe/atmospherics_gas_mask
+	name = "atmospherics gas mask"
+	result = /obj/item/clothing/mask/gas/atmos
+	tool_behaviors = list(TOOL_WELDER)
+	time = 8 SECONDS
+	reqs = list(
+		/obj/item/stack/sheet/mineral/metal_hydrogen = 1,
+		/obj/item/stack/sheet/mineral/zaukerite = 1,
+	)
```

Comment:
this seems sort of ridiculous on the face of it. it's a minor upgrade.
I don't think it warrents like, superpowers either.

Frankly the use of "the atmos metals" seems very uninspired, if you want it actually made as an upgrade, I think you should figure out something people would actually use, even if it's not like "atmos themed"

Reply:
oh it's not an upgrade it's a full recipe. that's kinda odd

Reply:
What's odd about it? The base gas mask is made of plastic. Plastic isn't known to be used as a very high quality and extremely durable material. The atmospherics gas mask can survive in a burn chamber with tritium as its fuel, or even a fusion chamber.

Reply:
just make it out of titanium or something, atmos-specific metals are never going to be made, especially considering you can get a modsuit version that also acts as a firesuit for much less

(atmos external plating)

Reply:
> What's odd about it?

It seems natural to me that an upgraded bit of equipment would be done as an upgrade to existing equipment.
Specially since it's done via crafting and not like a lathe recipe